python/humbug/report.py

```diff
@@ -237,6 +251,11 @@ def error_report(
         )
         if tags is None:
             tags = []
+
+        error_bases = self._extract_error_base(error=error)
+        for error_base in error_bases:
+            tags.append("error:{}".format(error_base))
```

Comment:
We don't need the base class of hte exception here. I suggest you do:
`tags.extend(["error:{}".format(error.__class__.name), "error:{}.{}".format(error.__module__, error.__class__.__name__)])`

Reply:
You can skip the first item (without `__module__`) if it makes sense.